client/flutter/ios/Runner/AppDelegate.swift

```diff
@@ -7,6 +7,9 @@ import Flutter
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
   ) -> Bool {
+    var flutter_native_splash = 1
```

Comment:
Where is this used?

Reply:
No clue, It doesn't really belong there, however, the package is putting it there.